spec/models/comfy/cms/user_spec.rb

```diff
@@ -1,4 +1,20 @@
 describe Comfy::Cms::User do
+  describe 'validations' do
+    describe 'role' do
+      it { should validate_presence_of(:role) }
+
+      it 'allows the role to be set as a "user"' do
```

Comment:
I would move this two íts to another describe block, since is not about validations.